packages/mvar/mvar.1.0.0/opam

```diff
@@ -18,6 +18,7 @@ remove: [
 ]
 depends: [
   "jbuilder" {build}
+  "opam-installer"
```

Comment:
I guess it's a `{build}` dependency too, right?

Reply:
That's a tough question. It's not used at build-time, it's used at install-time. What does the `{build}` modifier mean exactly? OPAM documentation doesn't say.

Reply:
It just means that reinstalling `opam-installer` will not trigger a recompilation of `mvar`. I guess it works for an intall-time dependency too.